data/db_demo.yaml

```diff
@@ -471,6 +472,16 @@ sources/ZTFrlh6cyjh/annotations:
 taxonomy:
   file: taxonomy_demo.yaml
 
+classification:
+  - taxonomy_id: 1
+    obj_id: ZTF18aabcvnq
+    probability: 0.9
+    classification: "Time-domain Source"
+  - taxonomy_id: 2
+    obj_id: ZTF18aabcvnq
+    probability: 0.02
+    classification: "Variable Taxonomy"
```

Comment:
I'm a little confused about the name "Variable Taxonomy". Wouldn't "Variable Classification" be more straightforward? Because we have both Taxonomy and Classification objects and this seems to be a Classification...

Reply:
Maybe simpler would be "Variable Star"?

Reply:
Yeah "Variable Star" is much better.

Reply:
Oh ok, I chose "Variable Taxonomy" because it was the first allowable class name for the taxonomy with id 2 in the demo data. I think the name of the classification has to match up with a class name from the taxonomy or the http request will fail with "That classification (Variable Star) is not in the allowed classes for the chosen taxonomy (id=2
make: *** [Makefile:59: load_demo_data] Error 255"

Reply:
Here's the possible class names, any preferences?
![Screenshot (7)](https://user-images.githubusercontent.com/82007190/147695613-9076bb08-9325-4ea4-a51f-d294ec2826d5.png)
![Screenshot (11)](https://user-images.githubusercontent.com/82007190/147695626-8a360c84-e171-413c-90b9-fa048e0f3e7f.png)

![Screenshot (9)](https://u
![Screenshot (10)](https://user-images.githubusercontent.com/82007190/147695618-365609d3-b9e5-4e49-b542-98566e92aa94.png)
ser-images.githubusercontent.com/82007190/147695616-60f39300-8948-4e70-9e58-ec8592040346.png)

Reply:
@jadalilleboe Ah "variable" then.

Reply:
@jadalilleboe Thanks for that. I guess the classification stuff has had no test to this point, so might be nice to take this opportunity to add one.